### DISCUSSION TITLE
FrankenWP with Roots Bedrock

### DISCUSSION THREAD

**Discussion by rdlphmchrd** [StephenMiracle/frankenwp#33] [Q&A]
Hi,
Do you think is it possible to use FrankenWP with [Roots Bebrock](https://roots.io/bedrock/) boilerplate instead of WordPress default structure?

**Comment by PascaleBeier** [comment]
Definitely so. I'll see if I can provide some documentation

**Reply by tislars** [reply]
Were you able to write this documentation? If so, where can we find it? I'd really like to be able to use this image, but without WordPress pre-installed. Instead, I'd prefer to install WordPress using Composer in the project.
